adminutils/adminutils.py

```diff
@@ -183,6 +183,62 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         else:
             await ctx.tick()
 
+    @emoji.command(name="steal")
+    async def emote_steal(self, ctx, name: str, message_id: int, *roles: discord.Role):
+        """
+        Steal an emoji from a message ID and add it to your server.
+        Role lock the emoji if you wish (use double quotes if role name has spaces)
+
+        Examples:
+            `[p]emoji steal thisReallyCoolName 743291269471797248`
+            `[p]emoji steal thisReallyCoolName 743291269471797248 RoleName`
+        """
+        channel = ctx.channel
+        message = await channel.fetch_message(message_id)
+        dig_up_emoji_check = message.content.split(":")[-3]
+        check_animated = dig_up_emoji_check.endswith("<a")
+        dig_up_emoji = message.content.split(":")[-1]
+        emoji = "".join(x for x in dig_up_emoji if x.isdigit())
+        check_animated = dig_up_emoji_check.endswith("<a")
+        if check_animated is True:
+            format_image = "gif"
+        else:
+            format_image = "png"
```

Comment:
This better to be redone with regex.

In example, as it done in [TrustyCogs' notsobot converter](https://github.com/TrustyJAID/Trusty-cogs/blob/a3e931bc6227645007b37c3f4f524c9fc9859686/notsobot/converter.py#L30-L36).